src/lib/api.js

```diff
@@ -409,15 +409,19 @@ const buildPaginationLinks = function (limit, parameters, bbox, intersects, endp
     const dictToURI = (dict) => (
       Object.keys(dict).map(
         (p) => {
-        // const query = encodeURIComponent(dict[p])
           let value = dict[p]
           if (typeof value === 'object' && value !== null) {
-            value = JSON.stringify(value)
+            if (p === 'sortby') {
+              const sortFields = []
+              for (let i = 0; i < value.length; i += 1) {
+                sortFields.push(value[i]['field'])
```

Comment:
what about direction?

Reply:
It will still work direction-wise (with one or multiple fields), and no code changes are needed there.

Reply:
Submitted another PR to handle this functionality.